pwa/app/routes/team.$teamNumber.tsx

```diff
@@ -105,10 +115,29 @@ export default function TeamPage(): JSX.Element {
           {team.city}, {team.state_prov}, {team.country}
         </InlineIcon>
 
-        <InlineIcon displayStyle={'flexless'}>
-          <BiInfoCircleFill />
-          {team.name}
-        </InlineIcon>
+        <Accordion type="single" collapsible>
+          <AccordionItem value="item-1" className="border-0">
+            <AccordionTrigger className="p-0 text-left font-normal">
+              <InlineIcon displayStyle={'flexless'}>
+                <BiInfoCircleFill />
+                {team.school_name ?? 'Family/Community'}
```

Comment:
I'm not sure this is something we can rely on, especially for older teams that went defunct before `school_name` was provided (e.g. 1717).